src/operations/cms/queries/getInternalNewsArticles.ts

```diff
@@ -1,12 +1,13 @@
 import { gql } from 'graphql-tag'
 
 export const GET_INTERNAL_NEWS_ARTICLES = gql`
-  query GetInternalNewsArticles($publishedDate: DateTime) {
+  query GetInternalNewsArticles($publishedDate: DateTime, $tag: String) {
```

Comment:
By adding $tag as an optional param, it allowed us to reuse the existing query and keep things DRY

Reply:
Thought: I wonder if we'll want to reuse this in the future or if there will be more drift in the queries. This works since it's low lift and we don't know how things will look once folks start using the api.

Reply:
This is good feedback. Paired with some other changes, I decided it's worth making this switch now and creating a new query under the thirdparty directory. Keep all the third-party code together, instead of getting too intertwined with existing code. Lmk if you have any concerns!